timeless/views.py

```diff
@@ -27,3 +27,57 @@ def get(self, *args, **kwargs):
             if result is None:
                 abort(404)
         return "", HTTPStatus.OK
+
+
+class GenericView(views.View):
+    """ Docs http://flask.pocoo.org/docs/1.0/views/ """
+    def dispatch_request(self):
```

Comment:
@vladarefiev I believe it's a not place for code of methods as described in docstring. Mb there will be permission checks and so on. See my text comment